tools/cli/commands.js

```diff
@@ -796,6 +796,11 @@ main.registerCommand({
   // the packages (or maybe an unpredictable subset based on what happens to be
   // in the template's versions file).
 
+  // Since some of the project skeletons include npm `devDependencies`, we need
+  // to make sure the current node environment is set to development (so those
+  // `devDependencies` are covered by `npm install`).
+  process.env.NODE_ENV = 'development';
```

Comment:
Interestingly enough `meteor create` will not install `devDependencies` from app skeleton `package.json`'s, as when run the `NODE_ENV` is `production` by default (which means `npm install` is called in production mode which by-passes `devDependencies`). 

This code change will temporarily change the `NODE_ENV` to `development` to make sure the `devDependencies` are picked up. Let me know if this should be handled differently. Thanks!

Reply:
Perhaps this could be a flag to `require("./default-npm-deps.js").install`, which could run `npm install --dev`?

Reply:
Sounds good to me - I'll adjust - thanks!

Reply:
Done @benjamn - quick FYI, it looks like `npm install --dev` has been deprecated and npm's recommended replacement of `--only=dev` doesn't do what we want here, since we want both `dependencies` and `devDependencies` to be installed. Luckily, npm has a wonderfully undocumented feature whereby the `--production` option can be passed `false` to override the current `NODE_ENV`, and install both `dependencies` and `devDependencies`. Go figure - thanks!